packages/legacy/core/App/hooks/proofs.ts

```diff
@@ -9,3 +12,16 @@ export const useProofsByConnectionId = (connectionId: string): ProofExchangeReco
     [proofs, connectionId]
   )
 }
+
+export const getAllCredentialsForProof = (proofId: string) => {
```

Comment:
is this function a hook (since it is in `hooks` folder)? If it is, it should be named `useAllCredentialsForProof`